### DISCUSSION TITLE
蔚蓝档案简介

### DISCUSSION THREAD

**Discussion by SilverWolf-8867** [share121/inter-knot#2524] [其他]
![Screenshot_20240810_113113](https://github.com/user-attachments/assets/5777aae6-78db-45ad-8836-ec297e335a12)
《蔚蓝档案》是一款由韩国‌NEXON Games旗下的‌MX studio开发的二次元角色扮演游戏，由‌上海悠星网络科技有限公司发行。该游戏于2021年2月4日上线日文版，海外版于2021年11月9日上线，简体中文版则在2023年8月3日上线。游戏的故事背景设定在一个名为“‌基沃托斯”的广阔学园都市，这个都市由数以千计的学校组成，形成了一个庞大而复杂的教育网络。在这个独特的都市中，学生们常常手持枪械，同学之间的冲突和纠纷也司空见惯。游戏的故事从‌联邦理事会长突然失踪开始，基沃托斯陷入混乱，玩家扮演的教师需要带领性格迥异的学生们通过重重考验，打败敌人，寻找真相。‌

《蔚蓝档案》的动画版《蔚蓝档案 The Animation》由‌Yostar Pictures和‌studio CANDYBOX负责制作，于2024年4月7日至6月23日在东京电视网各台依次播出，共12集，每集24分钟。‌

此外，《蔚蓝档案》的游戏玩法包括策略角色扮演、冒险等多种元素。玩家将扮演一名学院教师，与来自不同国家的学生们一起探索隐藏在蔚蓝档案背后的秘密，并为了守护世界的和平而展开一场惊心动魄的冒险。游戏中的学生角色有星级区分，最高为三星，玩家可以通过升级、装备和技能提升来增强角色的能力，打造个性化的战斗队伍。游戏的战斗系统采用了策略回合制的形式，玩家需要根据敌人的特点和关卡环境，合理安排队伍中角色的站位和技能释放顺序，以取得胜利。‌

**Comment by YuTinMIn** [comment]
看来我sensei的身份保不住了（

**Comment by AetherDrop** [comment]
你提到的游戏确实不错，但感觉不如原神。《原神》是由米哈游自主研发的一款全新开放世界冒险游戏。游戏发生在一个被称作「提瓦特」的幻想世界，在这里，被神选中的人将被授予「神之眼」，导引元素之力。你将扮演一位名为「旅行者」的神秘角色，在自由的旅行中邂逅性格各异、能力独特的同伴们，和他们一起击败强敌，找回失散的亲人——同时，逐步发掘「原神」的真相。

**Comment by blqdswkl** [comment]
sensei！どうしてここにいるんですか？

**Comment by lllliufeng** [comment]
谁问你了——啊，原来不是我

**Comment by Scottedy** [comment]
谁问你了

**Comment by 502awa** [comment]
ᕕ(◠ڼ◠)ᕗ

**Comment by niegong** [comment]
ᕕ(◠ڼ◠)ᕗ​

**Comment by yivitte** [comment]
ᕕ(◠ڼ◠)ᕗ​
